src/main/webapp/app/exercises/programming/assess/repo-export/programming-assessment-repo-export-dialog.component.ts

```diff
@@ -59,6 +61,10 @@ export class ProgrammingAssessmentRepoExportDialogComponent implements OnInit {
                 .pipe(
                     tap(({ body: exercise }) => {
                         this.exercise = exercise!;
+                        // TODO workaround since find does not support exam exercise permissions properly
+                        if (this.exercise.exerciseGroup?.exam?.course) {
+                            this.exercise.isAtLeastInstructor = this.accountService.isAtLeastInstructorInCourse(this.exercise.exerciseGroup?.exam?.course!);
+                        }
```

Comment:
```suggestion
                        // TODO workaround since find does not support exam exercise permissions properly
                        if (this.exercise.exerciseGroup?.exam?.course) {
                            this.exercise.isAtLeastInstructor = this.accountService.isAtLeastInstructorInCourse(this.exercise.exerciseGroup.exam.course!);
                        }
```
Nit: We can remove the question marks since we already did the check beforehand (Not sure if a exclamation mark is needed then, see below)